eng/targets/Helix.targets

```
@@ -97,6 +97,7 @@
     <PropertyGroup>
       <BuildHelixPayload Condition="'@(_HelixApplicableTargetQueue->Count())' == '0'">false</BuildHelixPayload>
       <BuildHelixPayload Condition="'$(TargetArchitecture)' == 'arm64' AND '$(SkipHelixArm)' == 'true'">false</BuildHelixPayload>
+      <BuildHelixPayload Condition="$(HelixTargetQueue.StartsWith('(Alpine.')) AND '$(SkipHelixAlpine)' == 'true'">false</BuildHelixPayload>
```

Might need '' around the condition
```suggestion
      <BuildHelixPayload Condition="'$(HelixTargetQueue.StartsWith('(Alpine.'))' AND '$(SkipHelixAlpine)' == 'true'">false</BuildHelixPayload>
```

---

I was thinking that too, but then I saw the following which apparently doesn't need to be surrounded by quotes despite being very similar.

https://github.com/dotnet/aspnetcore/blob/219ecd6880120215137f4d6ef0dd2aa62c780af2/src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets#L147

I wonder if we don't need to precent-encode the `(` though. Seems likely given the "Did you forget the closing parenthesis?" in the error.

```suggestion
      <BuildHelixPayload Condition="$(HelixTargetQueue.StartsWith('%28Alpine.')) AND '$(SkipHelixAlpine)' == 'true'">false</BuildHelixPayload>
```